src/e2e/java/teammates/e2e/pageobjects/GoogleLoginPage.java

```diff
@@ -1,38 +1,15 @@
 package teammates.e2e.pageobjects;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-
-import com.google.common.base.Preconditions;
-
-import teammates.e2e.util.TestProperties;
-
 /**
  * Page Object Model for the official Google Accounts login page.
  */
 public class GoogleLoginPage extends LoginPage {
```

Comment:
Actually this entire page can be deleted

Reply:
Let's keep it for now. I have a feeling we might be using this again some time; if I'm wrong we can always delete it later.